src/merge_jsons.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+"""
+This will merge 2 or more results.json files together to present in one "serve" report.
+
+This tool will replace keys in the merge, so you NEED unique names for EACH engines in EACH json.
+e.g. in json1 you could have the engine called "graviton2-tantivy-0.20" and in json2, "graviton3-tantivy-0.20".
+Same for other engines.
+
+Sample usage:
+    python3 src/merge_jsons.py --input_jsons foo.json bar.json more.json --output_json ./results2.json
+
+
+Dependencies:
+    * mergedeep
+
+pip3 install mergedeep to download this.
+"""
```

Comment:
Will this be part of usage if I run `python3 src/merge_jsons.py --help`? 

If not lets make it part of help message :)

Reply:
Good point, will do this. 
Will move some irrelevant parts (for printing) to a different comment.